src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -277,6 +285,8 @@ protected override async Task OnParametersSetAsync()
                 LoadLogs(newConsoleLogsSubscription);
             }
         }
+
+        this.PostComponentTelemetryParametersAsync();
```

Comment:
Is this hiding at the bottom of the method, beneath the local method? How come?

Reply:
There's no local method defined in that method. Because page state might be changing in OnParametersSet, it feels safer to always have this call at the end